test/math.jl

```diff
@@ -75,14 +75,13 @@ end
 @test_approx_eq erfinv(0.84270079294971486934) 1
 @test_approx_eq erfcinv(0.15729920705028513066) 1
 @test_approx_eq dawson(1) 0.53807950691276841914
-# TODO: complex versions only supported on 64-bit for now
-@unix_only if WORD_SIZE==64
-    @test_approx_eq erf(1+2im) -0.53664356577856503399-5.0491437034470346695im
-    @test_approx_eq erfc(1+2im) 1.5366435657785650340+5.0491437034470346695im
-    @test_approx_eq erfcx(1+2im) 0.14023958136627794370-0.22221344017989910261im
-    @test_approx_eq erfi(1+2im) -0.011259006028815025076+1.0036063427256517509im
-    @test_approx_eq dawson(1+2im) -13.388927316482919244-11.828715103889593303im
-end
+
+@test_approx_eq erf(1+2im) -0.53664356577856503399-5.0491437034470346695im
```

Comment:
should these be disabled on win32 for now, if it needs to wait until an llvm upgrade to pass there?
```
exception on 2: ERROR: LoadError: assertion failed: |erf(1 + 2im) - -0.536643565778565 - 5.049143703447035im| <= 8.881784197001252e-12
  erf(1 + 2im) = 9.588980733155008e-152 - 8.973262701968788e-29im
  -0.536643565778565 - 5.049143703447035im = -0.536643565778565 - 5.049143703447035im
  difference = 5.077581949584898 > 8.881784197001252e-12
 in error at error.jl:20
 in test_approx_eq at test.jl:137
 in anonymous at test.jl:145
 in test_approx_eq at test.jl:147
 in runtests at C:\projects\julia\test\testdefs.jl:78
 in anonymous at multi.jl:833
while loading math.jl, in expression starting on line 79
```